### DISCUSSION TITLE
is 'SetSceneItemEnabled' implemented in obs-websocket v5.0.0 ?

### DISCUSSION THREAD

**Discussion by kumarnalupu** [Palakis/obs-websocket#841] [Q&A] [answered]
Hi,

I am testing obs-websocket v5.0.0-alpha2 using simpleobsws v1.2.1 

Can you please let me know if 'SetSceneItemEnabled' request is implemented in this version or will come in next verson?

In the 'Scene Items' tab of the planned requests sheet that was posted [here](https://docs.google.com/spreadsheets/d/1LfCZrbT8e7cSaKo_TuPDd-CJiptL7RSuo8iE63vMmMs/edit#gid=1402923009) , it says 'SetSceneItemEnabled' request is implemented but when i try to execute it, i get error saying "Your request type is not valid.".

@tt2468

**Comment by tt2468** [comment, marked as answer]
It is implemented on the latest CI build available through the actions tab, but I'm not sure if it is included in alpha2.

**Reply by kumarnalupu** [reply]
It worked when I used latest CI build.
Thank you.
